src/components/atoms/NewTab/NewTab.styled.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const NewTabContainer = styled.div`
+  display: flex;
+  gap: 8px;
```

Comment:
Optionally, wouldn't just margin left and middle vertical align on icon be simpler?